app/notification/magic_link/map_contents.py

```diff
@@ -25,7 +25,7 @@ def from_json(magic_link_data):
             MagicLink object with magic link contents.
         """
         current_app.logger.info("Mapping contents for MAGIC_LINK")
-        data = MagicLink.process_data(magic_link_data)
```

Comment:
when does `process_data()` get called now then?

Reply:
This function calls the relevant service(MAGIC_LINK or APPLICATION) based on template type, makes it easier to understand I guess where template type makes a call to the API based on template type. Happy to go through when huddling.